### PR TITLE
Step 1 completed

### PR DESCRIPTION
**Pull request by jhileman07**

### CODE REVIEW

**Review by echavemann — CHANGES_REQUESTED** [review]
u need to reabse step-1-devvvv

**Comment by echavemann** [comment]
image

**Comment by echavemann** [comment]
also please better PR titles and close dead PRs

**Review by minnce — CHANGES_REQUESTED** [review]
initialize val in .cpp file please

File: src/pricingutil.cpp [hunk 1]
```diff
@@ -1,5 +1,14 @@
-//
-// Created by Ethan on 8/18/2023.
-//
-#include <pybind11/pybind11.h>
-#include "pricingutil.h"
+#include "pricingutil.h"
+
+PricingUtil::PricingUtil() {
```
Comment (minnce) [inline src/pricingutil.cpp]: PricingUtil::PricingUtil() : val(0.0) {}

File: src/pricingutil.cpp [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+  float val = 0;
```
Comment (minnce) [inline src/pricingutil.cpp]: remove this

**Review by minnce — CHANGES_REQUESTED** [review]

**Review by minnce — APPROVED** [review]
lgtm!

**Pull request merged**